components/scream/src/physics/spa/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+set(CLDFRAC_SRCS
+  atmosphere_prescribed_aerosol.cpp
+  atmosphere_prescribed_aerosol_hack.cpp
+)
+
+set(CLDFRAC_HEADERS
+  atmosphere_prescribed_aerosol.hpp
+  spa_functions.hpp
+  spa_functions_impl.hpp
+)
+
+add_library(spa ${CLDFRAC_SRCS})
+target_include_directories(spa PUBLIC ${CMAKE_CURRENT_SOURCE_DIR}/../share)
```

Comment:
You would need this only if you include stuff from physics/share without the "physics/share" part, like
```
#include "physics_constants.hpp"
``` 
We don't do that (and you don't do it in this PR, if I'm not mistaken). So you can/should remove this line.